galois_reed_muller.yml

```diff
@@ -0,0 +1,40 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# code id, physical, logical are all lower case
+code_id: galois_reed_muller
+physical: galois
+logical: galois
+
+name: 'Nonbinary Quantum Reed-Muller Codes'
+introduced: '\cite{doi:10.1109/18.771249}'
+
+description: 'The pure stabilizer code \([[q^m,k(v_2)-k(v_1),min{d(v_1^{\perp}),d(v_2)}]]_{q}\) with parameters \(k(v) = \sum_{j=0}^{m}(-1)^{j}\dbinom{m}{j}\dbinom{m+v-jq}{v-jq}\) and 
```

Comment:
We have a "true stabilizer code" entry, I think this is related or equal to "pure stabilizer code". Please verify, and list true stab as parent if so, and stab parent if not (adding a desciption of "pure").